x-pack/plugins/security_solution/cypress/integration/timelines/data_providers.spec.ts

```diff
@@ -33,6 +35,7 @@ describe('timeline data providers', () => {
     cleanKibana();
     loginAndWaitForPage(HOSTS_URL);
     waitForAllHostsToBeLoaded();
+    cy.scrollTo('bottom');
```

Comment:
Can this line of code be extracted to a method and put it on `x-pack/plugins/security_solution/cypress/tasks/common.ts`?